app_legacy/Helpers/database/user.php

```diff
@@ -199,42 +199,38 @@ function getUserPageInfo(string $user, int $numGames = 0, int $numRecentAchievem
 
     $libraryOut = [];
 
-    $libraryOut['RecentlyPlayedCount'] = getRecentlyPlayedGames($user, 0, $numGames, $recentlyPlayedData);
-    $libraryOut['RecentlyPlayed'] = $recentlyPlayedData;
     $libraryOut['MemberSince'] = $userInfo['Created'];
     $libraryOut['LastActivity'] = $userInfo['LastLogin'];
+    $libraryOut['LastActivityID'] = $userInfo['LastActivityID'];
     $libraryOut['RichPresenceMsg'] = empty($userInfo['RichPresenceMsg']) || $userInfo['RichPresenceMsg'] === 'Unknown' ? null : $userInfo['RichPresenceMsg'];
-    $libraryOut['LastGameID'] = $userInfo['LastGameID'];
-    if ($userInfo['LastGameID']) {
-        $libraryOut['LastGame'] = getGameData((int) $userInfo['LastGameID']);
```

Comment:
This game data lookup is now handled by the `withGameInfo` parameter of `getUserProgress`.